src/main/resources/swagger/api-docs.yaml

```diff
@@ -2731,6 +2731,54 @@ paths:
         200:
           description: OK
 
+  '/api/notifications/workspace/{workspaceNamespace}/{workspaceName}':
```

Comment:
paths and models should be organized alphabetically per https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/main/resources/swagger/README.md